h2o-docs/src/product/data-science/gam.rst

```diff
@@ -121,15 +282,49 @@ Common parameters
 - `stopping_rounds <algo-params/stopping_rounds.html>`__: Stops training when the option selected for ``stopping_metric`` doesn't improve for the specified number of training rounds, based on a simple moving average. To disable this feature, specify ``0`` (default). 
 
     **Note:** If cross-validation is enabled:
+
   
     - All cross-validation models stop training when the validation metric doesn't improve.
     - The main model runs for the mean number of epochs.
     - N+1 models may be off by the number specified for ``stopping_rounds`` from the best model, but the cross-validation metric estimates the performance of the main model for the resulting number of epochs (which may be fewer than the specified number of epochs).
 
+
+Example of specifying these parameters:
+
```

Comment:
Change

Example of specifying these parameters:

to 

Example of specifying GAM gridsearch hyperparameters: